### PR TITLE
calloc: fix order parameter

### PR DESCRIPTION
**Pull request by christophefontaine**
nit picking: First arg of calloc argument should specify number of elements, later size of each element.

### CODE REVIEW

**Comment by david-marchand** [comment]
I see 4 instances of this issue in origina/main.

$ git grep 'calloc(len'
modules/ip/control/address.c:   if ((resp = calloc(len, 1)) == NULL)
modules/ip/control/nexthop.c:   if ((resp = calloc(len, 1)) == NULL) {
modules/ip6/control/address.c:  if ((resp = calloc(len, 1)) == NULL)
modules/ip6/control/nexthop.c:  if ((resp = calloc(len, 1)) == NULL) {

**Review by rjarry — APPROVED** [review]

**Comment by david-marchand** [comment]
Just a note that in recent gcc version, this issue could be detected with -Wcalloc-transposed-args.

**Comment by david-marchand** [comment]
Changes lgtm.

File: meson.build [hunk 1]
```diff
@@ -19,17 +19,25 @@ project(
...
-add_project_arguments('-fstrict-aliasing', language: 'c')
-add_project_arguments('-Wstrict-aliasing=2', language: 'c')
 add_project_arguments('-Wno-format-truncation', language: 'c')
-add_project_arguments('-DGROUT_VERSION="' + meson.project_version() + '"', language: 'c')
+add_project_arguments('-Wstrict-aliasing=2', language: 'c')
+add_project_arguments('-fstrict-aliasing', language: 'c')
+
+# optional project C flags (keep alpha sorted)
+optional_c_args = [
+  '-Wmissing-variable-declarations',
```
Comment (christophefontaine) [inline meson.build:34]: Do you want to add `-Wcalloc-transposed-args` as well ?

Comment (rjarry) [reply meson.build:34]: done :+1:

**Pull request merged**